.github/actions/wait-for-es-data/action.yml

```diff
@@ -0,0 +1,35 @@
+name: "Wait for ES Data"
+description: "Waits for data on ES indices"
+inputs:
+  index:
+    description: "Which ES index to look at"
+    required: true
+
+runs:
+  using: composite
+  steps:
+    - name: Wait for data
+      env:
+        ES_HOST: http://localhost:9200
```

Comment:
It may be more robust to set es_host as an input variable with a default value of http://localhost:9200, and then use the following configuration:
```
env:
  ES_HOST: ${{ inputs.es_host }}
```